### PR TITLE
Playlist Page Styling Changes

### LINKED ISSUE #90: Style Spotify page to be consistent

### PR DESCRIPTION
**Pull request by courtneyeh**
**Problem**
Styling was not consistent with the rest of the web app.

**Changes Made**
- Styling uses the common AddButton
- Titles on playlist page are not consistent with others
- Empty, non-organiser playlist page is consistent

Closes #90

### CODE REVIEW

File: frontend/src/components/playlist/SpofityPlaylistPageWrapper/SpotifyPlaylistWrapper.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.title {
+    color: #24305e;
+    font-size: 30px;
+}
+
+.emptyPlaylistDescription {
+    font-style: italic;
```
Comment (Denijar) [inline frontend/src/components/playlist/SpofityPlaylistPageWrapper/SpotifyPlaylistWrapper.module.css:7]: can the description also have `color: #24305e;` to match the other pages

**Review by Denijar — CHANGES_REQUESTED** [review]
Nice styling changes look good! One lil colour change request. Also make sure @lhuijser gets a looksie :D

**Review by Denijar — APPROVED** [review]
Looks great to me

**Review by lhuijser — APPROVED** [review]
Changes look great to me! Thank you for doing this Courtney :D

**Pull request merged**